src/App/pages/GamesAdd/index.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import {
+  Row,
+  Col,
+  Card,
+  Form,
+  Button,
+  InputGroup,
+  FormControl,
+  DropdownButton,
+  Dropdown
+} from "react-bootstrap";
+
+import Aux from "../../../hoc/_Aux";
+
+class Dashboard extends React.Component {
```

Comment:
```suggestion
class GamesAdd extends React.Component {
```
Che me parece que se tiene que llamar asi el componente.